gslib/tests/test_acl.py

```diff
@@ -85,7 +85,7 @@ def test_set_invalid_acl_object(self):
                             return_stderr=True,
                             expected_status=1)
     if self._use_gcloud_storage:
-      error_text = 'Found invalid JSON/YAML file'
```

Comment:
Gsutil Output is something like;
`
Setting ACL on gs://<bucket_name>/...
ArgumentException: JSON ACL data could not be loaded from: badacl
`
gcloud storage output;
`
Updating gs://<bucket_name>/...                                      
`
`      
ERROR: Found invalid JSON/YAML in acl.txt 
`
`
JSON Error: Expecting value: line 1 column 1 (char 0)
  Completed 0                      
`  

`Because if it was the former, tests for this should always be failing ?` 
No gsutil tests are running with shim on. So none of test actually executed this condition and that's why it was not caught till now.

Reply:
Here also I meant parity tests. I think I will get the answer from an earlier comment, so will resolve this.